app/services/email_digest_article_collector.rb

```diff
@@ -51,11 +51,14 @@ def should_receive_email?
     return true unless last_email_sent_at
 
     # Has it been at least x days since @user received an email?
-    Time.current - last_email_sent_at >= Settings::General.periodic_email_digest
+    last_email_sent_at < Settings::General.periodic_email_digest.days.ago
```

Comment:
In scenarios like this I really prefer the `before?` and `after?` methods Rails adds:

```suggestion
    last_email_sent_at.before?(Settings::General.periodic_email_digest.days.ago)
```

Reply:
OMG, yes!  The mental gymnastics of `>` or `<` with dates is just a bit too much.

tl;dr - TIL about `.before?` and `.after?`

Reply:
this was 100% what I wanted - thanks for the tip